test/test_manifest.json

```diff
@@ -4863,6 +4863,12 @@
          "107R": {
            "value": false
          },
+    {  "id": "issue6127",
+       "file": "pdfs/issue6127.pdf",
+       "md5": "65ef9572bddef5fddfaf7172f10a0df2",
+       "rounds": 1,
+       "type": "eq"
+    },
```

Comment:
The placement of these lines caused the tests to fail, since the manifest file is not valid.

Also, it should suffice to test *only* the first page rather than the entire document; i.e. you want to add a `"lastPage": 1,` line to this block.

Reply:
oh, I just `node test/add_test.js ...` and I didn't check that's ok.